udata/core/dataset/models.py

```diff
@@ -324,7 +322,11 @@ def quality(self):
             * description length
             * and so on
         """
+        from udata.models import Discussion
```

Comment:
Yes, it's for avoid them in futur.
I can't help wondering it we are not missing a layer here. Having so much computation in a property might need action action layer.